frameworks/Java/sabina/jetty-mongodb.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+./setup.sh -Dsabina.backend=jetty -Dsabina.benchmark.repository=mongodb
```

Comment:
ping @msmith-techempower - FYI, you'll notice that the `&` is no longer required with the new way we call the setup files. Cool, huh? :-)